src/html.rs

```diff
@@ -94,6 +94,17 @@ pub fn get_attr(el: &NodeRef, attr_name: &str) -> Option<String> {
         .map(ToOwned::to_owned)
 }
 
+pub fn find_attr_name_bottom_up<'a>(el: &NodeRef, attr_names: &[&'a str]) -> Option<&'a str> {
```

Comment:
FWIW I don't think this is particularly a better name. I think `closest_attribute_in` or something would be ok. This matches the `.closest()` DOM API naming.

But I don't care much either way.